sdk/webpubsub/azure-messaging-webpubsubservice/azure/messaging/webpubsubservice/_models.py

```diff
@@ -0,0 +1,20 @@
+from typing import Optional, Any
+
+class GroupMember:
```

Comment:
Since return type of `list_connections_in_group` is `GroupMember`, you had better expose this class to SDK users. Then it is better to rename `_models.py` to `models.py`

Reply:
When I removed the underscore prefix of "_models.py", I got error like "ERROR:root:An error occurred while generating the coverage report for azure-messaging-webpubsubservice: No source for code: 'C:\Source\Repos\azure-sdk-for-python\sdk\webpubsub\azure-messaging-webpubsubservice\azure\messaging\webpubsubservice\_models.py'.    
INFO:root:Total coverage 0.0 for package azure-messaging-webpubsubservice" running `tox run -e whl", so I reverted the change. Looks like other services also use "_models" instead of "models"